core/kamon-core/src/main/scala/kamon/util/NoOpExecutorService.java

```diff
@@ -0,0 +1,108 @@
+package kamon.util;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.concurrent.*;
+
+/**
+ * An executor service that does nothing. Used with empty metrics so that no
+ * threads are created wasting time / space.
+ */
+public class NoOpExecutorService implements ScheduledExecutorService {
```

Comment:
I think, that `NoopModuleRegistry` and `NoOpExecutorService` should have the same prefix 😄

Reply:
fixed :)